cinnabar/tests/test_stats.py

```diff
@@ -145,3 +145,47 @@ def test_correlation_positive(fe_map):
         assert (
             0.5 < bss["mle"] < 0.9
         ), f"Correlation must be positive for this data. {stat} is {bss['mle']}"
+
+
+def test_confidence_intervals(fe_map):
+    """
+    Test that the bootstrapped confidence intervals
+    for RMSE and MUE contain the corresponding statistics.
+    Uses the example data in `cinnabar/data/example.csv`
+    """
+
+    nodes = fe_map.graph.nodes
+
+    x_data = np.asarray([n[1]["exp_DG"] for n in nodes(data=True)])
+    y_data = np.asarray([n[1]["calc_DG"] for n in nodes(data=True)])
+    xerr = np.asarray([n[1]["exp_dDG"] for n in nodes(data=True)])
+    yerr = np.asarray([n[1]["calc_dDG"] for n in nodes(data=True)])
+
+    # RMSE (default mode)
+    bss = bootstrap_statistic(x_data, y_data, xerr, yerr, statistic="RMSE")
+    assert bss['low'] < bss['mle'] < bss['high'], \
+        "The RMSE must lie within the bootstrapped 95% CI"
+
+    # MUE (default mode)
+    bss = bootstrap_statistic(x_data, y_data, xerr, yerr, statistic="MUE")
+    assert bss['low'] < bss['mle'] < bss['high'], \
+        "The MUE must lie within the bootstrapped 95% CI"
+
+    # RMSE (including xerr in bootstrapping)
+    bss = bootstrap_statistic(x_data, y_data, xerr, yerr, statistic="RMSE",
+                              include_true_uncertainty=True)
+    assert bss['low'] < bss['mean'] < bss['high'], \
+        "The RMSE must lie within the bootstrapped 95% CI"
+
+    # RMSE (including yerr in bootstrapping)
+    bss = bootstrap_statistic(x_data, y_data, xerr, yerr, statistic="RMSE",
+                              include_pred_uncertainty=True)
+    assert bss['low'] < bss['mean'] < bss['high'], \
+        "The RMSE must lie within the bootstrapped 95% CI"
+
+    # RMSE (including both xerr and yerr in bootstrapping)
+    bss = bootstrap_statistic(x_data, y_data, xerr, yerr, statistic="RMSE",
```

Comment:
> We're not testing passing statistic_type through
@IAlibay : That's because `statistic_type` is not an argument that's passed to `bootstrap_statistic`, it is only passed through `plotting.py` functions. We currently don't test any `plotting.py` functions, so would it be ok if I just opened an issue about writing tests for `plotting.py` as a long term action item?

Reply:
> Is our dataset such that these could ever fail? Can we add the dataset from the problematic example (i.e. https://github.com/OpenFreeEnergy/cinnabar/issues/73 ) to make sure we test against that edge case properly?

I added a test addressing the case in my latest commit. I hard coded the value directly into the test (which @dotsdl mentioned would be ok), since I was having trouble formatting the cinnabar input CSV to properly handle my dataset (previously described this problem here: https://github.com/OpenFreeEnergy/cinnabar/pull/74#discussion_r1073863391)

Reply:
>  We currently don't test any plotting.py functions, so would it be ok if I just opened an issue about writing tests for plotting.py as a long term action item?

Sorry that's on me, I should have realized that. Happy to go with just raising an issue about testing the plotting methods.